care/emr/api/viewsets/report/template.py

```
@@ -0,0 +1,281 @@
+            custom_types = FieldTypeRegistry.get_all()
+
+            schema = {
+                "contexts": contexts,
+                "output_formats": output_formats,
+                "custom_types": custom_types,
+            }
+
+            return Response(schema)
+
+        except Exception as e:
+            logger.exception("Failed to generate schema: %s", e)
```

Ruff (0.14.6)

Redundant exception object included in `logging.exception` call

(TRY401)